assets/components/linkerd2/values.yaml

```diff
@@ -0,0 +1,232 @@
+    crtExpiry:
+
+    # control plane annotation - do not edit
+    crtExpiryAnnotation: linkerd.io/identity-issuer-expiry
+
+    issuanceLifetime: 86400s
+
+    tls:
+      # PEM-encoded certificate
+      crtPEM: |
+
+      # PEM-encoded ECDSA private key
+      keyPEM: |
+
+# heartbeat configuration
+disableHeartBeat: false
+heartbeatSchedule: "0 0 * * *"
+
+# prometheus configuration
+prometheusImage: prom/prometheus:v2.15.2
+prometheusLogLevel: *controller_log_level
+prometheusExtraArgs: {}
+  # log.format: json
+prometheusAlertmanagers: []
+  # - scheme: http
+  #   static_configs:
+  #   - targets:
+  #     - "alertmanager.linkerd.svc:9093"
+prometheusRuleConfigMapMounts: []
+  # - name: alerting-rules
+  #   subPath: alerting_rules.yml
+  #   configMap: linkerd-prometheus-rules
+  # - name: recording-rules
+  #   subPath: recording_rules.yml
+  #   configMap: linkerd-prometheus-rules
+
+# proxy injector configuration
+proxyInjector:
+  externalSecret: false
+  # if empty, Helm will auto-generate these fields
+  crtPEM: |
+
+  keyPEM: |
+
+  # if empty, Helm will auto-generate this field, unless externalSecret is set to true.
+  caBundle: |
+
+# service profile validator configuration
+profileValidator:
+  externalSecret: false
+  # if empty, Helm will auto-generate these fields
+  crtPEM: |
+
+  keyPEM: |
+
+  # if empty, Helm will auto-generate this field, unless externalSecret is set to true.
+  caBundle: |
+
+# tap configuration
+tap:
+  externalSecret: false
+  # if empty, Helm will auto-generate these fields
+  crtPEM: |
+
+  keyPEM: |
+
+  # if empty, Helm will auto-generate this field, unless externalSecret is set to true.
+  caBundle: |
+
+# web configuration
+webImage: gcr.io/linkerd-io/web
+
+# If the namespace is controlled by an external tool or can't be installed with Helm
+# you can disable its installation. In this case:
+# - The namespace created by the external tool must match the namespace value above
+# - The external tool needs to create the namespace with the label:
+#     config.linkerd.io/admission-webhooks: disabled
+installNamespace: true
+
+# Node selection constraints for control-plane components
+# https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector.
+nodeSelector:
+  beta.kubernetes.io/os: linux
+
+smiMetrics:
+  enabled: false
+  externalSecret: false
+  image: deislabs/smi-metrics:v0.2.1
+  # if empty, Helm will auto-generate these fields
+  crtPEM: |
+
+  keyPEM: |
+
+  # if empty, Helm will auto-generate this field, unless externalSecret is set to true.
+  caBundle: |
+
+# Configuration for Add-ons
+
+grafana:
+  enabled: true
+  name: linkerd-grafana
+  image:
+    name: gcr.io/linkerd-io/grafana
+  # resources:
+
+tracing:
+  enabled: false
+  collector:
+    name: linkerd-collector
+    image: omnition/opencensus-collector:0.1.11
+    # resources:
+  jaeger:
+    name: linkerd-jaeger
+    image: jaegertracing/all-in-one:1.17.1
+    # resources:
```

Comment:
Can we please add assets and code in 2 separate commits, so it's easier to review?